src/other/tables.jl

```diff
@@ -82,8 +82,8 @@ Tables.schema(itr::Union{DataFrameRows, DataFrameColumns}) = Tables.schema(paren
 Tables.rowtable(itr::Union{DataFrameRows, DataFrameColumns}) = Tables.rowtable(parent(itr))
 Tables.namedtupleiterator(itr::Union{DataFrameRows, DataFrameColumns}) =
     Tables.namedtupleiterator(parent(itr))
-Tables.materializer(itr::Union{DataFrameRows, DataFrameColumns}) =
-    Tables.materializer(parent(itr))
```

Comment:
Any idea why we used this complex definition?

Reply:
Because we thought earlier that methods should be defined for instances not for types.
Also the old definition (with `parent`) tried to be more general in case someone would implement a custom `AbstractDataFrame`, but I do not think we need to be that careful. Currently - essentially - DataFrames.jl has the assumption that we only support `DataFrame` and `SubDataFrame` baked in in many places. Also falling back to `DataFrame` always is a relatively safe option.

Reply:
And we have `Tables.materializer(df::AbstractDataFrame) = DataFrame` anyway now, so mostly this does not change anything.